packages/lab/src/overlay/Overlay.css

```diff
@@ -1,93 +1,47 @@
-/* Experimental - Overlay css variable API */
 .saltOverlay {
-  --overlay-text-color: var(--salt-content-primary-foreground);
-  --overlay-spacing: var(--salt-size-unit);
-  --overlay-fontSize: var(--salt-text-fontSize);
-  --overlay-background: var(--salt-container-primary-background);
-  --overlay-borderColor: var(--salt-container-primary-borderColor);
-  /* TODO: z-index audit, provide z index here or rely on popper */
-  --overlay-zindex: var(--salt-zIndex-flyover);
+  --overlay-borderColor: var(--saltOverlay-borderColor, var(--salt-container-primary-borderColor));
+  --overlay-background: var(--saltOverlay-background, var(--salt-container-primary-background));
 }
 
-.saltOverlay-secondary.saltOverlay {
-  --overlay-background: var(--salt-container-secondary-background);
-  --overlay-borderColor: var(--salt-container-secondary-borderColor);
-}
+.saltOverlay {
+  font-family: var(--salt-text-fontFamily);
+  font-size: var(--saltOverlay-fontSize, var(--salt-text-fontSize));
+  font-weight: var(--saltOverlay-fontWeight, var(--salt-text-fontWeight));
+  line-height: var(--saltOverlay-lineHeight, var(--salt-text-lineHeight));
 
-.saltOverlay-content {
-  display: flex;
-  padding: calc(var(--overlay-spacing) * 2);
-  font-size: var(--overlay-fontSize);
+  border-color: var(--overlay-borderColor);
+  border-style: var(--saltOverlay-borderStyle, var(--salt-container-borderStyle));
+  border-width: var(--saltOverlay-borderWidth, var(--salt-size-border));
 
-  border: 1px solid var(--overlay-borderColor);
-  box-shadow: var(--salt-overlayable-shadow-popout);
-  line-height: var(--salt-text-lineHeight);
   background: var(--overlay-background);
-  color: var(--overlay-text-color);
-  position: relative;
-  opacity: 1;
-  overflow: visible;
-  z-index: var(--overlay-zindex);
-}
+  box-shadow: var(--saltOverlay-boxShadow, var(--salt-overlayable-shadow-popout));
 
-.saltOverlay-arrow,
-.saltOverlay-arrow::after {
-  border: 8px solid transparent;
-  position: absolute;
+  color: var(--saltOverlay-text-color, var(--salt-text-primary-foreground));
+  z-index: var(--saltOverlay-zIndex, var(--salt-zIndex-flyover));
+  width: max-content;
```

Comment:
Should we have a min/max width instead? what's the design direction?

Reply:
according to specs, height/width can be manually set by consumers to determine size as they wish. Let me double check with @joshwooding if we want to set a min width as Ben's OOO

cc @bhoppers2008